Framework/include/PrepNTupleVars.h

```diff
@@ -430,20 +436,21 @@ class PrepNTupleVars
         derivePtMassScaledJetAK8Collection(tr, jcAK8, "mpTScaled", 1.0, 1.0);
 
         // Create DeepCSV b-jet discriminator vector
-        const auto& Jets_bJetTagDeepCSVprobb       = tr.getVec<float>("Jets_bJetTagDeepCSVprobb");
-        const auto& Jets_bJetTagDeepCSVprobbb      = tr.getVec<float>("Jets_bJetTagDeepCSVprobbb");
-              auto& Jets_bJetTagDeepCSVtotb        = tr.createDerivedVec<float>("Jets_bJetTagDeepCSVtotb", Jets_bJetTagDeepCSVprobbb.size());
-        const auto& Jets_bJetTagDeepFlavourprobb   = tr.getVec<float>("Jets_bJetTagDeepFlavourprobb");
-        const auto& Jets_bJetTagDeepFlavourprobbb  = tr.getVec<float>("Jets_bJetTagDeepFlavourprobbb");
-              auto& Jets_bJetTagDeepFlavourtotb    = tr.createDerivedVec<float>("Jets_bJetTagDeepFlavourtotb", Jets_bJetTagDeepFlavourprobbb.size());
-        const auto& Jets_bJetTagDeepFlavourprobc   = tr.getVec<float>("Jets_bJetTagDeepFlavourprobc");
-        const auto& Jets_bJetTagDeepFlavourprobuds = tr.getVec<float>("Jets_bJetTagDeepFlavourprobuds");
-              auto& Jets_bJetTagDeepFlavourtotq    = tr.createDerivedVec<float>("Jets_bJetTagDeepFlavourtotq", Jets_bJetTagDeepFlavourprobc.size());
+        const auto& Jets_bJetTagDeepCSVprobb        = tr.getVec<float>("Jets_bJetTagDeepCSVprobb");
+        const auto& Jets_bJetTagDeepCSVprobbb       = tr.getVec<float>("Jets_bJetTagDeepCSVprobbb");
+              auto& Jets_bJetTagDeepCSVtotb         = tr.createDerivedVec<float>("Jets_bJetTagDeepCSVtotb", Jets_bJetTagDeepCSVprobbb.size());
+        const auto& Jets_bJetTagDeepFlavourprobb    = tr.getVec<float>("Jets_bJetTagDeepFlavourprobb");
+        const auto& Jets_bJetTagDeepFlavourprobbb   = tr.getVec<float>("Jets_bJetTagDeepFlavourprobbb");
+        const auto& Jets_bJetTagDeepFlavourproblepb = tr.getVec<float>("Jets_bJetTagDeepFlavourproblepb");
+              auto& Jets_bJetTagDeepFlavourtotb     = tr.createDerivedVec<float>("Jets_bJetTagDeepFlavourtotb", Jets_bJetTagDeepFlavourprobbb.size());
+        const auto& Jets_bJetTagDeepFlavourprobc    = tr.getVec<float>("Jets_bJetTagDeepFlavourprobc");
+        const auto& Jets_bJetTagDeepFlavourprobuds  = tr.getVec<float>("Jets_bJetTagDeepFlavourprobuds");
+              auto& Jets_bJetTagDeepFlavourtotq     = tr.createDerivedVec<float>("Jets_bJetTagDeepFlavourtotq", Jets_bJetTagDeepFlavourprobc.size());
 
         for(unsigned j = 0; j < Jets_bJetTagDeepCSVprobb.size(); ++j)
         {
             Jets_bJetTagDeepCSVtotb.at(j)     = ( Jets_bJetTagDeepCSVprobb.at(j) + Jets_bJetTagDeepCSVprobbb.at(j) );
-            Jets_bJetTagDeepFlavourtotb.at(j) = ( Jets_bJetTagDeepFlavourprobb.at(j) + Jets_bJetTagDeepFlavourprobbb.at(j) );
+            Jets_bJetTagDeepFlavourtotb.at(j) = ( Jets_bJetTagDeepFlavourprobb.at(j) + Jets_bJetTagDeepFlavourprobbb.at(j) + Jets_bJetTagDeepFlavourproblepb.at(j) );
```

Comment:
Do we only need to add this here? I'm wondering if all the variables that @cros0400 plans to use for inputs to the NN are defined well?

Reply:
From my understanding, here in `PrepNtupleVars.h` is where we are to define these "total" discriminator variables. In `MakeMVAVariables.h`, as an example, we then grab the newly registered collection of `"Jets"+myVarSuffix_+"_bJetTagDeepFlavourtotb"`

Reply:
Right. I meant does any other DeepFlavour variable need this added like the btag does? I know Bryan wants to use almost all of the DeepFlavour variables so I want to make sure we use them correctly.

Reply:
Ah, I see what you mean now. I will double check the docs.

Reply:
I guess the only other thing we could do would be to add the b disc components when constructing the `totq`. At the moment, seems that we have only been doing `probc + probuds = totq`. Though does @cros0400 use the `totq` variable ?

Reply:
I think he is using it. Let's add it so it is defined properly. I'm sure it is a minor change but no reason not to do it....I think. Let's make sure @cros0400 sees this.

Reply:
Yeah this would be the last variable. As for the totq, I wasn’t aware that the b jet component was missing. If the docs say to add it in then we should also add that.

Reply:
OK, I cannot find anything about `totq`, trying to remember when we added it---it might be something we came up with ourselves. I vaguely remember a discussion with @cmadrid1...maybe... 😄  The only explicit reference to any combination of the disc components is for the b disc.

Reply:
Alright, for now, I elect to leave the definition (our definition) of `totq` as is. The main update is to include `lepb` as part of the `totb`, which is a bit more official.